src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -6228,22 +6290,29 @@ static void addTie(const Notation& notation, Score* score, Note* note, const tra
         tie->setStartNote(note);
         tie->setTrack(track);
 
-        if (orientation == "over") {
-            tie->setSlurDirection(DirectionV::UP);
-        } else if (orientation == "under") {
-            tie->setSlurDirection(DirectionV::DOWN);
-        } else if (orientation == "auto") {
-            // ignore
-        } else if (orientation == "") {
-            // ignore
-        } else {
-            logger->logError(QString("unknown tied orientation: %1").arg(orientation), xmlreader);
+        const QColor color { notation.attribute("color") };
+        if (color.isValid()) {
+            tie->setColor(color);
```

Comment:
I believe adding `tie->setPropertyFlags(engraving::Pid::COLOR, engraving::PropertyFlags::UNSTYLED);` here would fix that strange partial import